Alarmi/Alarmi/Sources/Register/Supporting Views/TodayDdayView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  TodayDdayView.swift
+//  Alarmi
+//
+//  Created by LeeJiSoo on 2022/07/29.
+//  Copyright © 2022 MoTe. All rights reserved.
+//
+
+import UIKit
+
+protocol TodayDdayViewDelegate: AnyObject {
+    func presentCallDelayViewController()
+}
+
+final class TodayDdayView: UIView {
+
+    struct State {
+        var buttonName: String
+        var descriptionLabelName: String
+        var dDayLabel: String
+    }
+
+    // MARK: View
+
+    private let dDayStackView: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .vertical
+        $0.spacing = 7
+        $0.distribution = .fill
+        $0.alignment = .center
+        return $0
+    }(UIStackView())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        return $0
+    }(UILabel())
+
+    private let dDayLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.setDynamicFont(for: .largeTitle, weight: .bold)
+        $0.textAlignment = .center
+        return $0
+    }(UILabel())
+
+    private let callButton: UIButton = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.configuration?.baseBackgroundColor = .systemBlue
+        $0.configuration?.titleAlignment = .center
+        $0.configuration?.contentInsets = NSDirectionalEdgeInsets(top: 11, leading: 15, bottom: 15, trailing: 11)
```

Comment:
p1;

inset을 잘못 준 것 같습니다! 지금 top, trailing이 같고 leading, bottom이 같네요😝

```suggestion
        $0.configuration?.contentInsets = NSDirectionalEdgeInsets(top: 11, leading: 16, bottom: 11, trailing: 16)
```

Reply:
앗 캐치해주셔서 감사합니당! 꼼꼼아보 👍[407c187](https://github.com/DeveloperAcademy-POSTECH/MC3-Team7-MoTe/pull/97/commits/407c187926ec6599266fddad41d867a1fb0a5c34)